internal/test/examples.go

```diff
@@ -31,6 +31,9 @@ func ExampleEtcdCluster(namespace string) *v1alpha1.EtcdCluster {
 				},
 			},
 		},
+		Status: v1alpha1.EtcdClusterStatus{
+			Members:  make([]v1alpha1.EtcdMember, 0),
```

Comment:
related to my earlier comment about nil vs empty - are we sure that the value created here (empty) will behave the same as when it is created from the k8s manifests (where i'm assuming its nil)?

Reply:
I'm surprised that this needed to be changed. 
These examples get supplied to `client.Create` or `client.Update`, rather than `client.Status().Update`.
The API validation shouldn't require this to be set.
Unless I'm forgetting some other place where we use these examples.